proposals/dictionary-expressions.md

```diff
@@ -213,62 +274,68 @@ The new rules above represent a breaking change: For types that are a valid conv
 
 ## Create methods
 
-> A create method is indicated with a [CollectionBuilder(...)] attribute on the collection type. The attribute specifies the builder type and method name of a method to be invoked to construct an instance of the collection type.
-> ```diff
-> + A create method will commonly use the name `CreateRange` in the dictionary domain.
-> ```
-> For the create method:
->   - The method must have a single parameter of type System.ReadOnlySpan<E>, passed by value, and there is an identity conversion from E to the iteration type of the collection type.
->    - ```diff
->      + Or, the method have a single parameter of
->      + `IEnumerable<KeyValuePair<TKey, TValue>>` and iteration type
->      + of the collection type is the same `KeyValuePair<,>` type. 
-> ```
+~~A create method is indicated with a [CollectionBuilder(...)] attribute on the collection type. The attribute specifies the builder type and method name of a method to be invoked to construct an instance of the collection type.~~
+
+~~A create method will commonly use the name `CreateRange` in the dictionary domain.~~
+
+~~For the create method:~~
+   - ~~The method must have a single parameter of type System.ReadOnlySpan<E>, passed by value, and there is an identity conversion from E to the iteration type of the collection type.~~
```

Comment:
Why mark this section as deleted and replaced? It looks like it hasn't changed.

Reply:
Actually, the `CreateRange` statement is new to this proposal, so that should be bold, but the statements before and after appear unchanged.

Reply:
i thought you asked me to do that.  thi sis why i had a diff before.  :)

Reply:
my preference is just o write the new part, and have normal github diff be the diff mechanism.